lib/core.js

```diff
@@ -196,14 +196,18 @@ function Core(options) {
           Logger.getInstance().info('Configuring logger with level: %s, filePath: %s, additionalLogToConsole: %s', logLevel, logFilePath, additionalLogToConsole);
         }
 
-        const insecureConnect = options.insecureConnect;
-        if (Util.exists(insecureConnect)) {
+        if (Util.exists(options.insecureConnect)) {
+          Logger.getInstance().warn('Warning! The option insecureConnect will be deprecated. Please use the disableOCSPChecks.');
+        }
+
+        const disableOCSPChecks = options.insecureConnect || options.disableOCSPChecks;
```

Comment:
Are this values nullable? If yes and both are not null, `disableOCSPChecks` should have precedence, not the alternative.

Reply:
Changed.